From 7218145f925c81875d3616bf50d71e512329dee6 Mon Sep 17 00:00:00 2001
From: Ikey Doherty <ikey@solus-project.com>
Date: Wed, 12 Apr 2017 00:41:50 +0100
Subject: [PATCH 2/3] backends/native: Include sysmacros.h for major/minor
 macros

As of glibc 2.25, it is an error to use the old sys/types.h header for
the major/minor macros, and the correct sys/sysmacros.h header should be
used instead.

Signed-off-by: Ikey Doherty <ikey@solus-project.com>
---
 src/backends/native/meta-launcher.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/backends/native/meta-launcher.c b/src/backends/native/meta-launcher.c
index eb35f88..2d9de54 100644
--- a/src/backends/native/meta-launcher.c
+++ b/src/backends/native/meta-launcher.c
@@ -27,6 +27,7 @@
 #include <clutter/evdev/clutter-evdev.h>
 
 #include <sys/types.h>
+#include <sys/sysmacros.h>
 #include <sys/stat.h>
 #include <malloc.h>
 #include <fcntl.h>
-- 
2.14.2

